dolfyn/rotate/vector.py

```diff
@@ -148,6 +148,8 @@ def calc_omat(hh, pp, rr, orientation_down=None):
         #       orientation corresponds to the communication cable
         #       being up.  This is ridiculous, but apparently a
         #       reality.
+        #       In DOLfYN 'orientation_down' = True indicates the ADV probe head
+        #       is pointing UP.
```

Comment:
I think the correct text here is: "Nortek defines 'down' orientation as the ADV probe head points up."

Reply:
I didn't change anything here. It was confusing to me to try to sort out what 'orientation_down' in DOLfYN meant; I wasn't sure if it was still in Nortek defined terms or in down-means-down terms. I think the documentation needs a section clearly explaining some key details about ADVs without IMUs that are important to understanding DOLfYN plus a few links to more info in general ---  'orientation_down' is an output field from the binary file only from an ADV without an IMU, it'll = True if the ADV probe was headed up (Nortek Status bit 0 = 1) and = False (Nortek Status bit 0 = 0) if it the head was pointed down, that it only gets defined on the initial measurement in a time series, etc.. Reference the Transform.m file from Nortek as well as https://www.nortekgroup.com/faq/what-do-the-error-and-status-codes-mean, and Sven's 9/1/2009 posting on  http://www.nortek.no/en/knowledge-center/forum/current-profilers-and-current-meters/579860304 "You can read more about this in the manual, but what you need to be aware of is the update of the up/down orientation of the system. In order to calculate correct ENU coordinates there is an up/down sensor in the Vector as a part of the tilt sensor. And if you run continuous measurements this orientation is only established at the time when the deployment starts. "